components/compliance-service/habitat/default.toml

```diff
@@ -22,4 +22,4 @@ manual_polling_interval = 1440
 [agent]
 workers = 10
 buffer_size = 1000
-remote_inspec_version = "REPLACE-FROM-PLAN.SH"
+remote_inspec_version = "4.7.24"
```

Comment:
Just curious, why's this change here?

Reply:
This change should not be merged.

Reply:
fixing it, I am not sure how it got there, maybe from generated stuff?

Reply:
Yeah, it happens when you build the compliance-service hab package. I am really not sure why.